industrial_ci/src/tests/source_tests.sh

```diff
@@ -103,6 +109,8 @@ if [ "${USE_MOCKUP// }" != "" ]; then
     ln -s "$TARGET_REPO_PATH/$USE_MOCKUP" $CATKIN_WORKSPACE/src
 fi
 
+catkin config --install
```

Comment:
([My comment above](https://github.com/ros-industrial/industrial_ci/pull/150/files#r116532276) sounds a bit misleading, so I rephrase) With this line, users have no way to avoid testing against `install` space. E.g. https://github.com/ros-industrial-consortium/descartes/pull/198#issuecomment-301516448 where the tests depend on the files that are intentionally not deployed.